includes/class-wc-stripe-webhook-handler.php

```diff
@@ -503,10 +503,10 @@ public function process_webhook_refund( $notification ) {
 	 * @param object $notification
 	 */
 	public function process_review_opened( $notification ) {
-		$order = WC_Stripe_Helper::get_order_by_charge_id( $notification->data->object->charge );
+		$order = WC_Stripe_Helper::get_order_by_intent_id( $notification->data->object->payment_intent );
 
 		if ( ! $order ) {
-			WC_Stripe_Logger::log( 'Could not find order via charge ID: ' . $notification->data->object->charge );
+			WC_Stripe_Logger::log( 'Could not find order via intent ID: ' . $notification->data->object->payment_intent );
```

Comment:
The logged message is exactly the same here and below. Would that make debugging harder?

Reply:
I don't think it's a problem, because the webhooks are different (review opened and review closed), and the "closed" one only happens when the merchant takes a manual action in the Stripe dashboard.

Reply:
Will the logged messages have adjacent logged context as to which webhook was in play?

Reply:
No, it would just have a timestamp above the log message.

Reply:
Then I think we should add a short string to each with "review opened" and "review closed" or something like that.

Reply:
Done in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/908/commits/7ab522cff0cb91ed1db87b6784018e0090040a65